simplesamlphp-users.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
Please add a comment here:

```php
/*
  Specify user info for https://hub.docker.com/r/kristophjunge/test-saml-idp/ in the same format
  as our Seneca IdP will give it.  This is modeled on the original config here:
  https://github.com/kristophjunge/docker-test-saml-idp/blob/master/config/simplesamlphp/authsources.php
*/

Reply:
Why did this get resolved?  There's no comment yet.

Reply:
Oops - forgot to push. Had it commited.

Reply:
Its up now :)